From d48d62ccfffc941d44adc6847fb111fee49d46a7 Mon Sep 17 00:00:00 2001
From: Frank Lin <b33531@freescale.com>
Date: Mon, 16 May 2011 14:22:52 +0800
Subject: [PATCH 148/635] ENGR00143449 Fix system could not boot into UI issue

refine the code and fix bootup failture in alsa-lib

Signed-off-by: Frank Lin <b33531@freescale.com>
---
 modules/alsa/alsa_imx5x.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/modules/alsa/alsa_imx5x.cpp b/modules/alsa/alsa_imx5x.cpp
index d7e3d92..018c647 100755
--- a/modules/alsa/alsa_imx5x.cpp
+++ b/modules/alsa/alsa_imx5x.cpp
@@ -580,7 +580,6 @@ static status_t s_open(alsa_handle_t *handle, uint32_t devices, int mode)
 
     if (err == NO_ERROR) err = setSoftwareParams(handle);
 
-    setAlsaControls(handle, devices, mode);
 
     LOGI("Initialized ALSA %s device %s", stream, devName);
 
-- 
1.8.0

